Octokit/Models/Request/NewCommitCommentReaction.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class NewCommitCommentReaction
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="NewCommitCommentReaction"/> class.
+        /// </summary>
+        /// <param name="content">The reaction type.</param>
+        public NewCommitCommentReaction(EnumReaction content)
+        {
+            Ensure.ArgumentNotNull(content, "content");
```

Comment:
Actually you should be using [Paramater attributes](https://github.com/octokit/octokit.net/blob/f354d1bf00ff7606b46489e3a915e1428414bc47/Octokit/Helpers/ParameterAttribute.cs) on the enum value, which enables them to be automatically serialized.
[see here](https://github.com/octokit/octokit.net/blob/9231286fedce0c7a5f986d72cf1012a95941c62f/Octokit/Models/Request/MilestoneRequest.cs#L43) for an example